examples/basics/parameters/get-tabledata.js

```diff
@@ -0,0 +1,70 @@
+const createSession = require('../../session');
+
+// inline script with a simple table
+const script = `
+exampletable:
+load * Inline [
+dim, val
+foo, 23
+bar, 42
+c, 54
+d, 21
+];
+`;
+
+const session = createSession();
+
+/**
+ * Write the tabelContent to console.
+ * @param {*} tabelContent
+ */
+function print(tableContent) {
+  for (let i = 0; i < tableContent.length; i += 1) {
+    console.log(`dim: ${tableContent[i].qValue[0].qText} val: ${tableContent[i].qValue[1].qText}`);
+  }
+}
+
+// local variable to store the generated sessionapp
+let sessionApp;
+
+session.open()
+  .then(global => global.createSessionApp())
+  .then((app) => {
+    sessionApp = app;
+    return app.setScript(script);
+  })
+  .then(() => sessionApp.doReload())
+  .then(() =>
+    /**
+     * the getTableData functions receives the data from a specified Table.
+     * Since enigma.js v2.0.0 you can make the function call either by name or by position.
+     *
+     * The first example shows how to pass the parameter by position. By this way you insert
+     * the parameter in the order defined on the qlik help site for this function. Hereby the
```

Comment:
Instead of referring to the Qlik Help site, can we maybe refer to the QIX method itself? For example "... pass parameters by position in the order defined by the QIX method".